eng/versioning/version_data.txt

```diff
@@ -5,6 +5,7 @@
 com.azure:azure-data-sdk-parent;1.3.0;1.3.0
 com.microsoft.azure:azure-batch;11.0.0;11.1.0-beta.1
 com.microsoft.azure:azure-applicationinsights-query;1.0.0-Preview-1;1.0.0-Preview-2
+com.microsoft.azure:azure-applicationinsights-query-sample;1.0.0-Preview-1;1.0.0-Preview-1
```

Comment:
We generally don't include samples here.

Reply:
If I didn't add them the code analysis pipeline would complain about these POMs. Maybe it's a side effect of having the samples inherit from the Azure SDK Parent? Let me see if I can remove them without that happening.

Reply:
I'll keep the samples here. We do have sample Track 2 projects in the `version_client.txt` file: `azure-aot-graalvm-samples`, `azure-core-tracing-opentelemetry-samples`, `azure-identity-broker-samples`, and `azure-resourcemanager-samples`.